testdata/pkcs11-conf.json

```diff
@@ -0,0 +1,6 @@
+{
+    "module": "/usr/lib/libsofthsm.so",
```

Comment:
`integration/functions.sh:131` allows the module location to be overridden by setting `$PKCS11_MODULE` before calling `log_prep_test()`. However, it looks like this config file will remain hardcoded to "/usr/lib/libsofthsm.so". Will the override actually work if this file isn't also changed?